### PR TITLE
316896-update-dashboardlisting-content-in-readme-file

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,8 @@ This project was created using ASP.NET Core 6.0. This application aims to demons
   
   3. Finally, run the application using the command `dotnet run`. After the application has started, it will display a URL in the `command line interface`, typically something like (e.g., https://localhost:5001). Copy this URL and paste it into your default web browser.
 
+> **NOTE:** By default, the dashboard listing is hidden in the sample. If you want to render a dashboard with a specific listing of dashboards, you need to add a suburl to the web address and retrieve the dashboard listing accordingly.(e.g., https://localhost:5001/dashboardlisting).
```
Comment (vinothkumar1732) [inline README.md]: We represent the dashboard embedding by default without the dashboards listing sidebar. You must navigate to the "dashboardlisting" URL (such as https://localhost:5001/dashboardlisting) to enable the dashboards list.

File: README.md [hunk 2]
```diff
@@ -57,6 +59,8 @@ This project was created using ASP.NET Core 6.0. This application aims to demons
 
     ![dashboard image](https://github.com/boldbi/aspnet-core-sample/assets/91586758/4af68f49-ffc0-400a-a323-55a3f3600a1d)
 
+> **NOTE:** By default, the dashboard listing is hidden in the sample. If you want to render a dashboard with a specific listing of dashboards, you need to add a suburl to the web address and retrieve the dashboard listing accordingly.(e.g., https://localhost:5001/dashboardlisting).
```
Comment (vinothkumar1732) [inline README.md]: We represent the dashboard embedding by default without the dashboards listing sidebar. You must navigate to the "dashboardlisting" URL ( such as https://localhost:5001/dashboardlisting) to enable the dashboards list.

File: README.md (hunk 1 above)
Comment (vinothkumar1732) [reply README.md]: Update the port number and URL for other two platform samples.

**Review by vinothkumar1732 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md]: Content updated

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: Content updated

**Review by vinothkumar1732 — COMMENTED** [review]
Change is fine.

**Pull request merged**